src/app.tsx

```diff
@@ -22,12 +21,13 @@ export default class App extends React.Component<{}, AppState> {
   async componentDidMount() {
     // Fetch data from database
     const response = await axios.get<AppState>("/data");
-    const [headers, ...data] = response.data.data;
+    const [headers, ...raw] = response.data.data;
```

Comment:
Hmm... I also like pulling out the first row server-side and serving a response with `{ headers: string[], data: string[][]`

Reply:
Sorry, but this is not what I meant. Since now I am not using `headers` variable as a map key for width and column header, we don't really need extra array that only contains `headers`.

Reply:
Oh, yes! I see what you mean now.

Yeah, you can just strip the headers completely then (either client or server-side).